### ISSUE TITLE
MacOS > VoiceOver > VO keys > provide option grouping of elements 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14136]
All cards components 

Grouping of cards and possibility to navigate them with VO + arrow keys or with Quick Nav turned on 

Description: This issue is related to the “Primary method of navigation using VoiceOver on macOS“ remark at the top of this document. The cards should be implemented as groups that can be interacted with, and pressing of the VO key + the arrow keys should therefore work to navigate between the cards. 
Severity: high. 

What we want to reach: 
User would navigate through the cards in the same way as he navigate for example though toolbars. It means VO+left/right arrow keys would navigate into the card and then user would need press Control-Option-Shift-Down arrow to navigate inside the card.

Quick investigation show 
https://developer.apple.com/library/archive/documentation/Accessibility/Conceptual/AccessibilityMacOSX/EnhancingtheAccessibilityofStandardAppKitControls.html
Paragraph: Define links and groups to provide context.
Not sure how to implement "NSAccessibilityGroup" protocol. Or how to progress with this. 
But would be nice for future to decide for VoiceOver users, which elements could be groupped.
